.github/workflows/trivy-nightly-scan.yaml

```diff
@@ -0,0 +1,34 @@
+name: Trivy Nightly Scan
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+
+jobs:
+  nightly-scan:
+    name: Trivy Scan nightly
+    strategy:
+      fail-fast: false
+      matrix:
+        # maintain the versions of etcd that need to be actively
+        # security scanned
+        versions: [v3.4.22, v3.5.6]
```

Comment:
Is it possible to automatically scan any new versions? 

Since 3.4.22 and 3.5.6 are already released, so there will be no any changes on them. I think one time scanning should be OK. In other words, no need to scan then every day. Did I miss anything?

Reply:
Hi @ahrtr , we can automatically scan new versions if we get a `latest` tag for the new versions. I think we need to scan daily or on a specific cadence since new CVEs can get reported everyday.

Reply:
I recall that @serathius intentionally removed the `latest` tag sometime ago. 



> I think we need to scan daily or on a specific cadence since new CVEs can get reported everyday.

Based on this, it seems that it isn't necessary to have a `latest` tag either.  What's your preferred scan frequency? @ptabor @serathius @spzala  Daily seems OK to me.

Reply:
We need to update the versions each time when we release new etcd versions. Correct?

Reply:
Yes correct.

Reply:
Also, are images created with every build? If yes, should we have a separate workflow which builds from latest commit and does a trivy scan(daily)?

Reply:
I thought about it, and basically it looks good to me. Probably we can have weekly build. Please feel free to raise an issue to get feedback firstly.